packages/next/build/swc/crates/core/tests/errors.rs

```diff
@@ -0,0 +1,59 @@
+use next_swc::disallow_re_export_all_in_page::disallow_re_export_all_in_page;
+use std::fs::read_to_string;
+use std::path::{Path, PathBuf};
+use swc::Compiler;
+use swc_ecmascript::visit::Fold;
+use swc_ecmascript::{
+    parser::{Syntax, TsConfig},
+    transforms::pass::noop,
+};
+use testing::{fixture, run_test};
+
+#[fixture("tests/errors/re-export-all-in-page/**/input.js")]
+fn re_export_all_in_page(input: PathBuf) {
+    test_stderr(&input, disallow_re_export_all_in_page(true));
+}
+
+#[allow(unused_must_use)]
+fn test_stderr(input: &Path, transform: impl Fold) {
```

Comment:
This might be useful https://docs.rs/swc_ecma_transforms_testing/0.41.0/swc_ecma_transforms_testing/fn.test_fixture_allowing_error.html

Reply:
Oh nice, thanks!